swarm.cabal

```diff
@@ -154,35 +155,26 @@ library swarm-lang
       -- See discussion in #415
       StrictData
 
-library swarm-engine
+library swarm-scenario
```

Comment:
As part of this PR we should also rerun the `render-sublibrary-dependencies` script and check in an updated SVG.